content/project-news/reactos-0415-released.md

```diff
@@ -0,0 +1,98 @@
+---
+title:       "ReactOS 0.4.15 released"
+author:      "ReactOS Team"
+date:        2025-03-21
+tags:        [ "release" ]
+banner:      "img/project-news/reactos-0415-released/thumb.png"
+---
+
+We are pleased to announce the release of ReactOS 0.4.15!
+This release offers USB boot fixes, audio fixes, memory management fixes, registry healing, improvements to accessories and system tools including Notepad, Paint, the Input Method Editor, and shell improvements.
+
+We chose to release this version of ReactOS in honor of Eric Kohl's [first commit](https://git.reactos.org/?p=reactos.git;a=commit;h=108fcceee22216b169b448932bb3f86c81b9448e) to the ReactOS code base, which dates back to the 1999 year.
+Eric Kohl is the oldest active contributor of the project, and this is his 26th ReactOS anniversary!
+He is participating in the project as a developer basically since its beginnings.
+
+![Eric Kohl's commit activity on GitHub](/img/project-news/reactos-0415-released/EricKohl.png)
+
+This release is a culmination of the work of countless contributors since 0.4.14 was branched in 2020.
+This has been the largest release to date.
+There are nearly 8 times more commits going into this release than in 0.4.14.
+We are proud of the progress we have made, and are eager to continue with this growth.
+The pace of development for ReactOS has been growing exponentially.
+Let's dive in and see what's new.
+
+## USB Boot
+Victor Perevertkin (Extravert-ir) has accomplished major code cleanup and refactoring for better compatibility with Windows drivers and USB booting.
+(PnP and I/O Kernel stack) **NEEDS TO BE FINISHED**
+
+Johannes Anderwald (janderwald) solved an issue where the USB driver would enter an infinite loop when a USB device would not enter the ready state. Fixing this infinite loop allowed ReactOS to boot on more hardware.
+
+## Audio
+Thanks to the work of Oleg Dubinskiy, 0.4.15 features many audio improvements. Oleg added support for more audio formats, looped playback of wave files, higher sample rates, and multiple output channels. In addition, Victor Perevertkin imported the open source AC97 driver from the Windows Driver Kit (WDK). This enables sound out of the box in VirtualBox when the virtual machine is configured to use the ICH AC97 Audio Controller.
+
+## Memory Manager and Cache Controller
+Section Objects have been refactored by Jérôme Gardou (zefklop) for better compatibility with Windows. This fixes a long-standing bug preventing executables from starting in remote locations, such as network shares or virtual machine shared folders.
+
+Because of improvements in the memory manager and cache controller, we can now import the open source Microsoft FAT filesystem driver from the Windows Driver Kit (WDK). This FAT filesystem driver is a huge improvement from the old FAT filesystem driver which was slower and less stable. Additionally, external drives with FAT filesystem can now be properly ejected thanks to this new driver.
```

Comment:
```suggestion
Because of improvements in the memory manager and cache controller, we can now import the open source Microsoft FAT filesystem driver from the Windows Driver Kit (WDK). This FAT filesystem driver is a huge improvement from the old one, which was slower and less stable. Additionally, external drives with FAT filesystem can now be properly ejected thanks to this new driver.
```